interface/forms/dictation/new.php

```diff
@@ -1,29 +1,51 @@
-<!-- Form generated from formsWiz -->
 <?php
+/** 
+ *  Dictation Form
+ * 
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @copyright Copyright (c) 2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * 
+ */
 
-
-
+use OpenEMR\Core\Header;
 
 include_once("../../globals.php");
 include_once("$srcdir/api.inc");
 formHeader("Form: dictation");
 $returnurl = 'encounter_top.php';
 ?>
-<html><head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<html>
+<head>
+  <title>Dictation</title>
+   <?php Header::setupHeader(); ?>
+   <script type="text/javascript" src="<?php echo $webroot."/public/assets/ckeditor-4-7-0/ckeditor.js" ?>"</script>
+   <script src="<?php echo $webroot."/public/assets/ckeditor-4-7-0/js/sample.js" ?>" type="text/javascript"></script>
+   <link href="<?php echo $webroot."/public/assets/ckeditor-4-7-0/css/samples.css"; ?>" rel="stylesheet" type="text/css" />
```

Comment:
For the above 3 lines, need to use the global for the assets path(this will allow us to migrate that path easily if needed):
``````
<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/public/assets/ckeditor-4-7-0/ckeditor.js"</script>
<script src="<?php echo $GLOBALS['assets_static_relative']; ?>/public/assets/ckeditor-4-7-0/js/sample.js"  type="text/javascript"></script>
<link href="<?php echo $GLOBALS['assets_static_relative']; ?>/public/assets/ckeditor-4-7-0/css/samples.css"  rel="stylesheet" type="text/css" />